[2022-06-27 13:30:48,527] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-27T00:00:00+00:00 [queued]>
[2022-06-27 13:30:48,538] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-27T00:00:00+00:00 [queued]>
[2022-06-27 13:30:48,539] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:48,541] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:30:48,541] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:48,583] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-01-27 00:00:00+00:00
[2022-06-27 13:30:48,602] {standard_task_runner.py:52} INFO - Started process 56875 to run task
[2022-06-27 13:30:48,630] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-01-27T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp0vbfxs0e', '--error-file', '/tmp/tmpnovoepvn']
[2022-06-27 13:30:48,632] {standard_task_runner.py:80} INFO - Job 128: Subtask model_A
[2022-06-27 13:30:48,739] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-01-27T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:30:49,047] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-01-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-27T00:00:00+00:00
[2022-06-27 13:30:49,049] {python.py:173} INFO - Done. Returned value was: 7
[2022-06-27 13:30:49,207] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220127T000000, start_date=20220627T133048, end_date=20220627T133049
[2022-06-27 13:30:49,336] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:30:49,560] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
